src/views/Clients/ViewClients/index.jsx

```diff
@@ -29,11 +34,33 @@ import clientsQuery from './clients.graphql';
     ...theme.mixins.fab,
   },
 }))
-export default class ViewWorker extends PureComponent {
+export default class ViewClients extends PureComponent {
   state = {
     clientSearch: '',
+    previousClientId: '',
```

Comment:
You can remove the error with:

```jsx
// eslint-disable-next-line react/no-unused-state
previousClientId: '',
```

It's also done in https://github.com/taskcluster/taskcluster-web/blob/04d3822572f7bbb6ae831ad6edbe9e5ad2c61898/src/components/ClientForm/index.jsx#L99.